tests/integration/mutation/delete/field_kinds/one_to_many/with_show_deleted_test.go

```diff
@@ -68,25 +38,36 @@ func TestDeletionOfADocumentUsingSingleDocIDWithShowDeletedDocumentQuery(t *test
 			},
 			testUtils.CreateDoc{
 				CollectionID: 1,
-				Doc:          jsonString1,
+				Doc: `{
+					"name": "John",
+					"age": 30
+				}`,
 			},
 			testUtils.CreateDoc{
 				CollectionID: 0,
-				Doc:          jsonString2,
+				DocMap: map[string]any{
+					"name":      "John and the philosopher are stoned",
+					"rating":    9.9,
+					"author_id": testUtils.NewDocIndex(1, 0),
+				},
 			},
 			testUtils.CreateDoc{
 				CollectionID: 0,
-				Doc:          jsonString3,
+				DocMap: map[string]any{
+					"name":      "John has a chamber of secrets",
+					"rating":    9.9,
+					"author_id": testUtils.NewDocIndex(1, 0),
+				},
 			},
 			testUtils.Request{
-				Request: fmt.Sprintf(`mutation {
-					delete_Book(docID: "%s") {
+				Request: `mutation {
+					delete_Book(docID: "bae-b5c56d8f-b2f5-57f9-b371-4e9e04903e91") {
```

Comment:
question: Is it necessary to hardcode the docID here? You're doing the effort to switch to `NewDocIndex` in a lot of places. It seems like these would also be good candidates for that.